src/tools/gxhttpd/opt.ss

```diff
@@ -0,0 +1,119 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; The Gerbil HTTP Daemon
+;;;
+(import :std/cli/getopt
+        :std/config)
+(export #t)
+
+(def server-config-option
+  (option 'config "-c" "--config"
+    help: "location of the httpd configuration; when running as a standalone server it defaults to $GERBIL_PATH/httpd/config; when running as part of an ensemble this option is ignored"))
+
+(def ensemble-config-option
+  (option 'config "-c" "--config"
+    help: "location of the httpd ensemble configuration; it defaults to $GERBIL_PATH/ensemble/config"))
```

Comment:
Are the flags intentionally similar?

Reply:
yes, they are not used concurrently, and i want a contextual help.